app/controllers/organizations_controller.rb

```diff
@@ -79,6 +79,18 @@ def reactivate_user
     redirect_to user_update_redirect_path, notice: "User has been reactivated."
   end
 
+  def opt_in_email_notification
```

Comment:
This should just be a field on the organization edit page - it doesn't really deserve its own route.

Reply:
@dorner Sir, I didnt add these lines however i remove reactived and deactived redirect paths.